action_msgs/srv/CancelGoal.srv

```diff
@@ -11,5 +11,23 @@
 # Goal info containing an ID and timestamp
 GoalInfo goal_info
 ---
+# Response codes
+# Indicates the request was accepted.
+# One or more goals have transitioned to the CANCELING state.
+# The goals_canceling list should not be empty.
+int8 RESPONSE_OK=0
+# Indicates the request was rejected.
+# No goals have transitioned to the CANCELING state.
+# The goals_canceling list should be empty.
```

Comment:
Consistency: here it says "should be empty" while for the next response code it says "is empty".

Reply:
b5d202d

Reply:
+ 62eceb4